### PR TITLE
Fix key input on multiple properties page

### PR DESCRIPTION
**Pull request by rwfeather**
## Change description

The new help text below the input didn't behave as I expected once someone else got their hands on it. I was using the state variable where I shouldn't have.

This fix makes the text more static, and prevents the text from showing up when a user makes edits to already acceptable keys.

Check Brian's recording to see the problems this fixes: https://share.getcloudapp.com/5zunBQAm

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx [hunk 1]
```diff
@@ -191,11 +191,12 @@ export default function Page(props) {
               onChange={(e) => setKey(e.target.value)}
               disabled={disabled}
             />
-            {!exactProperty && key !== column ? (
+            {!exactProperty &&
+            columnSpeculation[column].suggestedPropertyKey !== column ? (
```
Comment (edmundito) [inline ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx]: `columnSpeculation[column].suggestedPropertyKey` could be set as a const var at top of the function so it's not too wordy in the component rendering.

Comment (rwfeather) [reply ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx]: Thanks! Definitely looks cleaner

**Review by edmundito — APPROVED** [review]

**Review by teallarson — APPROVED** [review]
Looks good!  It would be helpful to put a screenshot of the change in the description since it's a UI-related PR.
